netlify.toml

```diff
@@ -2,7 +2,11 @@
 [[headers]]
   for = "/*"
   [headers.values]
-    Content-Security-Policy = "default-src https:; script-src https: 'unsafe-inline'; style-src https: 'unsafe-inline'; img-src https: data: blob:; frame-ancestors *"
+    Content-Security-Policy = "default-src 'self'; script-src 'self' https: 'unsafe-inline'; style-src https: 'unsafe-inline'; img-src https: data: blob:; frame-src data:; frame-ancestors *; object-src 'none'; connect-src https:;"
```

Comment:
What's the point of `frame-src data:;`? What iframe is that meant to accommodate? ChatGPT called it out as being uncommon:

![image](https://github.com/rilldata/rill/assets/14206386/6ae89574-4f94-4659-b8a4-80a3b751aa52)

Reply:
Was getting `Refused to frame '' because it violates the following Content Security Policy directive`
https://serverfault.com/questions/945834/refused-to-frame-because-it-violates-the-following-content-security-policy-di suggested `data:` and it worked!